sphinx/source/docs/user_guide/export.rst

```diff
@@ -1,46 +1,49 @@
 .. _userguide_export:
 
-Exporting Plots
+Exporting plots
 ===============
 
-Additional Dependencies
+Additional dependencies
 -----------------------
 
-In order to use the |export| functions, users may have to install additional
-dependencies. One option is to install them via conda:
+You may need additional dependencies to use the |export| functions.
+You can install these dependencies in several ways, including the
+following:
+
+• Using Conda:
```

Comment:
They might render in a way, because Sphinx just puts the asci character into the html code. However, this is not a good way to generate lists with Sphinx - you should definitely use asterisks as described in the link to the spinx docs.

Reply:
Right, sorry I was not specific, I meant would they be rendered as actual `<li>` items as an asterisk would be

Reply:
Okay, noted. Assumed it would be okay to use this character because of how [bullet lists](https://docutils.sourceforge.io/docs/ref/rst/restructuredtext.html#bullet-lists) are described in the reStructuredText Markup Specification.

Reply:
Using asterisks also ensures a certain level of consistency throughout the various elements of Bokeh's docs. There are often different ways to do one and the same thing with ReST. When in doubt, I would generally follow the recommendations in the Sphinx documentation.